LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -4,59 +4,61 @@
 
 namespace LearningMissionSimulation
 {
-    public class PlaygroundMher
+    public class PlaygroundMher : ISimulation
     {
-        public PlaygroundMher()
-        {
-            
-        }
-       
-        //Account account = ObjectGenerator.GenerateAccount();
+        List<Account> studentList = new List<Account>();
+
+        List<Account> instructorList = new List<Account>();
 
         List<Account> accountList = new List<Account>();
 
-        Queue<Status> statusQueue = new Queue<Status>();
+        Queue<Account> accountQueueList = new Queue<Account>();
```

Comment:
The name "accountQueueList" does not make any sense. You're declaring and creating a Queue, why are you calling it a list?! Please rename to accountQueue.